mobify/platform-scaffold
# Fetch saved addresses when initializing the checkout shipping page.

The Salesforce Connector wasn't set up to fetch saved addresses when arriving on the checkout shipping page. Due to this none of the saved addresses would show up unless you visited the Account saved addresses page. 

<img src="https://user-images.githubusercontent.com/77138/30718796-f9605d7c-9ed5-11e7-8bce-acfbee35e12e.png" width="320px">

 **JIRA**: N/A
 **Linked PRs**: N/A

## Changes
- Described above

## Todos

## How to test-drive this PR
- Run `npm start`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fmobify-tech-prtnr-na03-dw.demandware.net%2Fon%2Fdemandware.store%2FSites-2017refresh-Site%2FHome-Show&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Login to an account that has at least one saved address
- Add an item to the cart
- Begin checkout
- Verify that saved addresses are listed on the Checkout **Shipping** page (as shown in screenshot above)
